controls/def.cf

```diff
@@ -94,6 +94,22 @@ bundle common def
                            " allowallconnects in body server control if not",
                            " already specified" );
 
+    !disable_always_accept_policy_server_allowallconnects::
+      "control_server_allowallconnects_derived" -> { "ENT-10951" }
+        slist => { @(def.control_server_allowallconnects), "$(sys.policy_server)" },
+        comment => concat( "We want to define the default setting for",
```

Comment:
Kind of feel like the comment should mention including `$(sys.policy_server)` or not including it below.

Reply:
I see you fixed this in a later commit. 👍